src/dialects/snowflake/query-generator-typescript.ts

```diff
@@ -13,4 +17,32 @@ export class SnowflakeQueryGeneratorTypeScript extends AbstractQueryGenerator {
     // TODO [+snowflake-sdk]: check if this is the correct implementation
     return `SELECT '' FROM DUAL`;
   }
+
+  removeIndexQuery(tableName: TableNameOrModel, indexNameOrAttributes: string | string[], options: RemoveIndexQueryOptions) {
+    // TODO [+snowflake-sdk]: check if this is the correct implementation
```

Comment:
Do you have any reason to doubt this implementation?

Reply:
Looking through the snowflake docs, I can't find any reference to indexes.
From a brief search, it looks like snowflake doesn't support indexes however I haven't used dialect so can't confirm 100%.

Reply:
Did a quick Google search and you seem to be correct; https://popsql.com/learn-sql/snowflake/how-to-drop-an-index-in-snowflake
Let's just leave it like this and once we can run integration tests on snowflake people can fix it

Reply:
Okay, I've changed my mind. I think we should remove the implementation of `removeIndexQuery` for snowflake. The abstract QueryGenerator will throw an error and that's fine imo.

Reply:
Ok, I've removed this now